### ISSUE TITLE
Accessibility for copilot experience

### ISSUE THREAD

**Issue by psugasani** [microsoft/sample-app-aoai-chatGPT#198]
I was working on the sample app to create a copilot experience that can answer questions on HPC and IaC. Everything has gone great so far and I've even expanded the citations to display more metadata from the files. However, I shifted focus to the accessibility of the app and I'm having trouble moving focus from a citation generated to the citations panel onClick/onPress. I was hoping for some insight on possible steps to make this change happen.

**Comment by davidm00** [comment]
Hi @psugasani ,

 Can you provide more information on the transition specifically? On my end, I can still tab into the citation panel. I was hoping you can provide repro steps to better address the issue :). 
 
 Thanks

**Comment by psugasani** [comment]
Hi @davidm00,

The issue is not tabbing access to the citation panel, this part works great, and I want to thank the team for adding keyboard access to the citation panel. What I'm exploring is after a user presses enter/clicks on a citation generated, the keyboard highlight should be moved directly to the citation panel. So that the user's next tab press should enter the citation panel. Currently after a user presses/clicks on a citation the keyboard highlight does not automatically move to the citation panel and instead stays within the response generated. To then reach the citation panel the user has to use tab a few number of times. As a user I expect the keyboard highlight to be moved automatically to the content that is most relevant to the most recently clicked button. Another place a change could be implemented would be the stop generating button. Where there is no alert to the user that they can stop generating content. For me the flow that makes the best sense in this case, is that when a user sends a questionInput, the keyboard highlight is moved automatically to the stop generating button, until the answer is generated.

**Comment by davidm00** [comment]
I see, thanks for the feedback. I will look into this :)

**Comment by psugasani** [comment]
Thank you!
